tests/requirements/py3.txt

```diff
@@ -8,6 +8,7 @@ numpy
 Pillow >= 6.2.0
 # pylibmc/libmemcached can't be built on Windows.
 pylibmc; sys.platform != 'win32'
+pymemcache >= 3.3.0
```

Comment:
Why did you choose `3.3.0`? It looks that everything will work properly with 3.0, or even less.

Reply:
That's not true. I fixed a number of issues with API compatibility in https://github.com/pinterest/pymemcache/pull/292. The intent was to avoid needing hacks in Django. By restoring the hacks we can reduce this to `>= 3.0.0`, but I don't think it's worth it.

I've also just noticed that `HashClient` doesn't properly support UNIX sockets - it assumes `(host, port)` 2-tuples - so I'm trying to address that in https://github.com/pinterest/pymemcache/pull/295.

Reply:
Thanks for details. Should we wait for more fixes in `pymemcache` before we move forward this patch?

Reply:
It might be worth it. I may take a stab at sorting out IPv6 support as that works with `pylibmc`.

I'll apply all your other changes now and I'll let you know when everything has settled down with changes in `pymemcache`.

Reply:
Thanks :+1:

Reply:
I've pinned this at `>= 3.4.0` as that version includes working support for IPv6 and UNIX sockets.